DM-42606: Update registry sections to reflect new schema registry configuration (#2)
lsst-dm/dmtn-210: index.rst

This is done through a Kubernetes "Job", which is a set of instructions to run certain commands inside a container in the Kubernetes cluster. That Job is defined in `sync-schema-job.yaml`_.

The schema synchronization job is a wrapper around a script in the `lsst/alert_packet`_ repository.

> The script is named ``syncAllSchemasToRegistry.py``, and its name says it all: it syncs all schemas into the `lsst/alert_packet`_ repository and submits it to a Schema Registry.

Reviewer: ```suggestion
The script is named ``syncAllSchemasToRegistry.py``, and its name describes its function: it syncs all schemas contained in the `lsst/alert_packet`_ repository to the Schema Registry.
```

> As all schemas are within the registry, the repository must be recreated each time, however the schemas will always be assigned the same id.

Reviewer: ```suggestion
Due to the details of how the Schema Registry handles schemas, it must be recreated from scratch each time.  However, the synchronization script ensures that the uploaded schemas will always be assigned the same id.
```

This value is passed in as the ``schemaSync.image.tag`` value when configuring the `alert-stream-schema-registry`_ chart.
Note that this version is probably **not** the version of the Alert Packet Schema that will be synchronized since the version of the alert_packet repository is independent from that of the schemas.

If syncing the registry does not trigger a refresh of the Docker image, the Docker ``digest`` can be passed to ``schemaSync.image.digest``` which can force a refresh of the Docker image.

When the Job runs
*****************

> The Job runs whenever the full alert-stream-broker Phalanx service is synchronized in Argo CD, not when induvidual components are synced.

Reviewer: ```suggestion
The Job runs whenever the full alert-stream-broker Phalanx service is synchronized in Argo CD, not when individual components are synced.
```

This means that it is run on essentially any change to any of the components of the entire Alert Distribution System, not just when the alert packet schema changes.

> This is perhaps unnecessarily often, but no changes to the schema will be made as the schema ids are manually assigned. The schema registry

Reviewer: ```suggestion
This is perhaps unnecessarily often, but no user-facing changes will be apparent as the schema ids are manually assigned. The schema registry
```
